test/pushgatewayTest.js

```diff
@@ -96,6 +96,19 @@ describe.each([
 						expect(err.message).toStrictEqual('Pushgateway request timed out');
 					});
 			});
+
+			it('should be possible to configure for gravel gateway integration (no job name required in path)', async () => {
+				const mockHttp = nock('http://192.168.99.100:9091')
+					.post('/metrics', body)
+					.reply(200);
+
+				instance.requireJobName = false;
```

Comment:
instead of mutating `instance`, create a new one?

Reply:
Wanted to do that, but for some reason I failed to get passing tests on my first try. Just committed my second try.